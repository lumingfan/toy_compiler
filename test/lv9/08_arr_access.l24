int ga[10] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};

int main() {
    int i = 0;
    int sum = 0;
    while (i < 10) {
        sum = sum + ga[i];
        i = i + 1;
    }
    putint(sum);
    putch(10);

    i = 0;
    int a[10] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
    while (i < 10) {
        sum = sum + a[i];
        i = i + 1;
    }
    putint(sum);
    putch(10);
    return 0;
}
