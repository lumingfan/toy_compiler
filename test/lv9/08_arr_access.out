55
110
0
